src/main/java/graphql/schema/idl/errors/NotAnInputTypeError.java

```diff
@@ -0,0 +1,12 @@
+package graphql.schema.idl.errors;
+
+import graphql.language.TypeDefinition;
+
+import static java.lang.String.format;
+
+public class NotAnInputTypeError extends BaseError {
+
+    public NotAnInputTypeError(TypeDefinition typeDefinition) {
+        super(typeDefinition, format("expected InputType, but found %s type", typeDefinition.getName()));
```

Comment:
Have  a look at the other errors.  They try to include a linecol() of the offending definition